src/dashboards.js

```diff
@@ -166,27 +203,91 @@ function getHumanReadableOptions(filterName, options, os) {
         }
         return ((a < b) ? -1 : ((a > b) ? 1 : 0));
       }).map(function(option) {
-        return [os + "," + option, osName + " " + (windowsVersionNames.hasOwnProperty(option) ? windowsVersionNames[option] : option), osName];
+        return [os + "," + option, osName + " " + (windowsVersionNames.hasOwnProperty(option) ? windowsVersionNames[option] : option), "Any " + osName];
       });
     } else if (os === "Darwin") {
       return options.map(function(option) {
         for (var prefix in darwinVersionPrefixes) {
           if (option.startsWith(prefix)) {
-            return [os + "," + option, osName + " " + option + " (" + darwinVersionPrefixes[prefix] + ")", osName];
+            return [os + "," + option, osName + " " + option + " (" + darwinVersionPrefixes[prefix] + ")", "Any " + osName];
           }
         }
-        return [os + "," + option, osName + " " + option, osName];
+        return [os + "," + option, osName + " " + option, "Any " + osName];
       });
     }
-    return options.map(function(option) { return [os + "," + option, osName + " " + option, osName]; });
+    return options.map(function(option) { return [os + "," + option, osName + " " + option, "Any " + osName]; });
   } else if (filterName === "arch") {
     return options.map(function(option) {
       return [option, archNames.hasOwnProperty(option) ? archNames[option] : option];
     });
   } else if (filterName === "measure") {
-    return options.map(function(option) { return [option, option] });
+    return options.sort().map(function(option) { return [option, option] });
   } else if (filterName === "channelVersion") {
-    return options.map(function(option) { return [option, option.replace("/", " ")] });
+    // Find the latest nightly version
+    var latestNightlyVersion = 0;
+    options.forEach(function(option) {
+      var parts = option.split("/");
+      if (parts[0] === "nightly" && parseInt(parts[1]) > latestNightlyVersion) {
+        latestNightlyVersion = parseInt(parts[1]);
+      }
+    });
+    
+    // Sort the options in the specified order and set aside the bad ones
+    var badOptions = [], goodOptions = [];
+    options.sort(function(a, b) {
+      var parts1 = a.split("/"), parts2 = b.split("/");
+      if (channelVersionOrder.hasOwnProperty(parts1[0]) && channelVersionOrder.hasOwnProperty(parts2[0])) {
+        if (channelVersionOrder[parts1[0]] !== channelVersionOrder[parts2[0]]) {
+          return channelVersionOrder[parts1[0]] - channelVersionOrder[parts2[0]];
+        }
+        return parseInt(parts2[1]) - parseInt(parts1[1]);
+      }
+      if (channelVersionOrder.hasOwnProperty(parts1[0])) { return -1; }
+      if (channelVersionOrder.hasOwnProperty(parts2[0])) { return 1; }
+      if (parts1[0] > parts2[0]) { return 1; }
+      if (parts1[0] < parts2[0]) { return -1; }
+      return parseInt(parts2[1]) - parseInt(parts1[1]);
+    }).forEach(function(option) {
```

Comment:
split this into two separate statements, this is getting a bit too long to be readable